src/components/Images/Images.js

```diff
@@ -2,13 +2,7 @@ import React from 'react'
 import PropTypes from 'prop-types'
 import useRequired from '../../hooks/useRequired.js'
 import useInvalid from '../../hooks/useInvalid.js'
-
 import { createCloudinaryLegacyURL } from '@cloudinary/url-gen'
-
-import { v4 as uuidv4 } from 'uuid'
```

Comment:
during testing I realized that we cannot generate an unique uuid as the key for every `img` element on the page, as it causes React to re-request the same image during scrolling of the page since they have different keys.

Reply:
furthermore, there are shorter unique values we could use for making component keys unique per build. We( don't need the level of uniqueness that uuidv4 provides, which is:

> For there to be a one in a billion chance of duplication, 103 trillion version 4 IDs must be generated.